UnityProject/Assets/Scripts/Objects/WallmountSpriteBehavior.cs

```diff
@@ -36,9 +36,11 @@ private void OnWillRenderObject()
 		{
 			return;
 		}
+		//Allows getting parent's assumed position if inside object
+		objectBehaviour = PlayerManager.LocalPlayer.GetComponent<ObjectBehaviour>();
```

Comment:
This code won't compile, you forgot to define a variable.
But most importantly you **really** don't want to call `GetComponent` for each wallmount every frame!
instead refer to local player script where there's already a reference to PushPull (and ObjectBehaviour is its subclass)
So just access it instead and don't forget to change `pushPull` field type in PlayerScript to ObjectBehaviour as well
```suggestion
				var objectBehaviour = PlayerManager.LocalPlayerScript.pushPull;
```